paper/writeup.tex

```diff
@@ -1237,12 +1232,31 @@ \section{Discussion}
 used a heuristic algorithm to produce ``plausible'' ARGs
 (i.e., those that explained the data with few mutations and recombinations),
 and searched for associations with traits in the resulting ensemble of ARGs.
+(See \citet{wong2024general} for more historical discussion.)
 Our approach for extending haplotypes follows the same logic,
 that an ARG with fewer recombination events is more parsimonious,
 and thus more likely.
 For this reason, it will occasionally be wrong even if the trees are correct,
 although in practice this source of error is likely much smaller than error in tree inference itself.
 
+\paragraph{IBD in ARGs}
+The term ``identity by descent'' (IBD) is used to mean many different (but related) things,
+and length distributions of shared IBD segments can be used for inference of recent demographic history
+\citep[for instance,][]{alasadi2018estimating,ringbauer2017inferring,browning2015accurate}.
```

Comment:
We should probably cite the other definitions of ARG IBD also, like e.g.
```
One definition of IBD in the ARG context is to consider how the most recent common ancestor of two nodes changes along the genome~\citep{carmi2016registering,huang2024estimating}. Another commonly used...
```

- Rapidly registering identity-by-descent across ancestral recombination graphs. Journal of Computational Biology. 2016;23(6):495–507.
- Estimating evolutionary and demographic parameters via  ARG-derived IBD (accepted in PLOS genetics). https://www.biorxiv.org/content/10.1101/2024.03.07.583855v2.full